docs/troubleshooting.adoc

```diff
@@ -12,7 +12,10 @@ This can be due to a number of reasons:
   * If in a repo, ensure the walkthrough is in the `walkthroughs` directory of the
   repo. 
 - The repo containing the walkthrough in `WALKTHROUGH_LOCATIONS` was updated after the server has started.
-  * If the repo was updated after the server has started, invoke the `POST /sync-walkthroughs` endpoint or restart the server.  
+  * If the repo was updated after the server has started, invoke the `POST /sync-walkthroughs` endpoint, for example `curl -X POST localhost:3006/sync-walkthroughs`.
++
+NOTE: This sync feature only works for local development, it does not work when the web app is hosted on OpenShift. When necessary restart or redeploy the web app.
```

Comment:
@pb82 Does this sound right to you?

I'm wondering why this is the case though. A technical limitation?

Reply:
No technical limitation, we intentionally disable that: https://github.com/integr8ly/tutorial-web-app/blob/master/server.js#L104

But thinking about it, it would be useful to allow this even in production.

Reply:
> But thinking about it, it would be useful to allow this even in production.

Agreed. But we should think about it more before adding it in for production.
Some things to consider:

* auth
* debouncing
* throttling